be/src/runtime/fragment_mgr.cpp

```diff
@@ -248,31 +246,17 @@ Status FragmentExecState::execute() {
     return Status::OK();
 }
 
-Status FragmentExecState::cancel_before_execute() {
-    // set status as 'abort', cuz cancel() won't effect the status arg of DataSink::close().
-#ifndef BE_TEST
-    SCOPED_ATTACH_TASK(executor()->runtime_state());
-#endif
-    _executor.set_abort();
-    _executor.cancel();
-    if (_pipe != nullptr) {
-        _pipe->cancel("Execution aborted before start");
```

Comment:
where to cancel this pipe now?

Reply:
Status FragmentExecState::cancel(const PPlanFragmentCancelReason& reason, const std::string& msg)  this method will cancel pipe.